packages/ag-charts-website/src/pages/debug/interface-models/[interfaceName].json.ts

```diff
@@ -1,28 +1,13 @@
-import { buildModel } from '@features/api-documentation/utils/model';
-import { getJsonFromDevFile } from '@utils/getJsonFromDevFile';
+import { buildModel } from '@features/api-explorer/utils/model';
+import { getDeprecateLookupFiles } from '@utils/devFiles';
 import { getModelInterfaces } from '@utils/pages';
-import { execSync } from 'child_process';
-
-interface Params {
-    interfaceName: string;
-}
 
 export async function getStaticPaths() {
     return getModelInterfaces();
 }
 
-export async function get({ params }: { params: Params }) {
-    const { interfaceName } = params;
-    const interfaceLookup = getJsonFromDevFile('ag-charts-community/interfaces.AUTO.json');
-    const codeLookup = getJsonFromDevFile('ag-charts-community/doc-interfaces.AUTO.json');
+export async function get({ params: { interfaceName } }: { params: { interfaceName: string } }) {
+    const { interfaceLookup, codeLookup } = getDeprecateLookupFiles();
```

Comment:
I guess we can either delete this dev page or update it to the models you generate

Reply:
Technically the generated file today if much more readable, so not sure if there's need for this page, but if you wanna keep it anyhow so we'll have a web interface for it we could easily point it to the new implementation instead.
Let me know what you prefer.

Reply:
I prefer to delete it, if it's redundant 🔥 😄